src/js/constants/contentConstants.js

```diff
@@ -1 +1 @@
-export const aboutUrl = '/public/about.html';
+export const aboutUrl = '/public/';
```

Comment:
Nitpick: 'might be worth renaming `aboutUrl` to `partialAboutUrl` for consistency

Reply:
Good idea!